### DISCUSSION TITLE
Tables & Fields created in Database do not reflect in app

### DISCUSSION THREAD

**Discussion by absolutegravitas** [directus/directus#17478] [Archive]
### Describe the Bug

Table & Field I create in a postgreql database (in Supabase) do not show in Directus app.

### To Reproduce

1 - on postgres database create new table
2 - add 2 fields any type
3 - commit
4 - refresh directus app
Result - new table is shown in Directus Data Model as collection ✔️ 
Result - fields are shown in new table collection ❌ 

5 - attempt to create new field on above collection via Directus
6 - Result - field is created on the collection ❌ Error Message shown

### Errors Shown

no
![image](https://user-images.githubusercontent.com/124488796/217806870-7d9d3078-f777-48c2-8b90-478fde849fed.png)
![image](https://user-images.githubusercontent.com/124488796/217807027-d6b697d2-6fdd-4426-9783-6194acdc6b63.png)

![image](https://user-images.githubusercontent.com/124488796/217806899-33f79db7-06c3-45a1-b0f5-66fd681e3cdf.png)

### What version of Directus are you using?

9.22.3

### What version of Node.js are you using?

18.0.0

### What database are you using?

Postgres 13

### What browser are you using?

Firefox

### How are you deploying Directus?

docker on fly.io

**Comment by absolutegravitas** [comment]
Also attempted with 9.18.x, 9.22.1 with no effect... 
Also attempted with CACHE_SCHEMA=false
is this a postgres thing?

**Reply by aypm-dev** [reply]
I'm having the same error with mySQL with directus 9.22.3

**Reply by aypm-dev** [reply]
Is there a reason why this was converted into a discussion instead of keeping it as an issue?

Quote from the docs: 

```Database Tables

Remember, a collection is simply a database table. Therefore, you can import or create a table directly in the database and it will automatically appear within your Directus project. The first time you manage that table, a directus_collections record will be created with default values.```

**Comment by MentalGear** [comment]
I'm also looking into using Supabase with Directus. What's the current state of integration?
Supabase lists Directus as a partner but Direcuts docs don't have any mention of supabase.
https://supabase.com/partners/directus

**Reply by absolutegravitas** [reply]
hey @MentalGear -- turns out my original issue was due to permissions on postgresql tables that weren't correctly set at the time of migration -- had to export table definitions via sql statements instead of using the dBeaver out of the box migrate / move feature

once I resolved that and migrated data, it was a matter of refreshing the definitions inside Directus with a quick edit of any given field.

unfortunately I had to do this table by table as FK / PK relationships aren't quite maintained either and Directus gets confused by them as well.
